.github/workflows/lint.yml

```diff
@@ -0,0 +1,39 @@
+name: Lint
+
+on: [push, pull_request]
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/pip
+            ~/.cache/pre-commit
+          key: lint-v1-${{ hashFiles('**/pyproject.toml') }}-${{
+            hashFiles('**/.pre-commit-config.yaml') }}
+          restore-keys: |
+            lint-v1-
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U pre-commit
+
+      - name: Lint
+        run: pre-commit run --all-files --show-diff-on-failure
```

Comment:
Let's use pre-commit/action instead? It's simpler.

Reply:
I hadn't seen that one before, much better! Updated!